### ISSUE TITLE
Behaviour of Voice Over when marking text

### ISSUE THREAD

**Issue by aazgin** [microsoft/vscode#142163]
When marking Text in Code using Voice Over, a Marked Block gets read from the beginning whenever something is added to the block.
So say if I've got the text "foo bar" and start marking it by pressing shift +arrow right from "f", instead of reading "f marked" "o marked" voice over reads "f" marked "fo" marked" "foo marked" and so on. While this might be intended for a single line, this also happens when marking text line by line. Getting only the last line as spoken output should be enough.

**Comment by isidorn** [comment]
@aazgin thanks for filling this. 
VS Code is built on top of Electron which uses Chromium, so let's see if this is actually a Chromium issue. Can you reproduce the same problem on this page https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea please try it in the textArea "It was a dark and stormy night"

If you can not reproduce in Chromium, than can you reproduce with this VS Code build https://az764295.vo.msecnd.net/exploration/50cb341a51e538e4f4beb574dd50a93546e5b0da/VSCode-darwin-universal.zip

**Comment by aazgin** [comment]
Thanks @isidorn 
I can't reproduce the behaviour with the first link, but with the VSCode-Version in the second link, it is the same.

**Comment by isidorn** [comment]
@aazgin thank you that is very useful. How about if you go to vscode.dev, can you reproduce then in your Chrome browser?
https://vscode.dev/github/microsoft/vscode-docs

**Comment by isidorn** [comment]
@aazgin thanks! When you say marking text, do you just mean selecting text in VS Code? Or is there something else here?

Because I just tried to reproduce the issue you are describing and I could not. But I am not a VoiceOver user so I might be missing something here. VO never reads "marked" for me, is that some special feature?

**Comment by aazgin** [comment]
@isidorn sorry, of course I mean selecting. :D

**Comment by isidorn** [comment]
@aazgin ok, so I tried to repro and I get the same behaviour in the [textArea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) and in VS Code using VoiceOver.
Let's say I have the following text and I am selecting character by character:
```
It was a dark and stormy night...
```

When "It was" is selected VoiceOver reads:
"It wa, unselected It was, selected"


Now I find this VoiceOver behaviour weird because "it was" is actually selected so what is read is really confusing.
I can file this bug to VoiceOver, but can you please:
1. Confirm if this is exactly what you are hearing in this case?
2. Clarify what is the difference in behaviour in [textArea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) or in VS Code. Because I do not see a difference

**Comment by isidorn** [comment]
Let's reopen to give more time for @aazgin to reply
